charts/navidrome/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for FreshRSS.
+
+image:
+  repository: deluan/navidrome
+  pullPolicy: IfNotPresent
+  tag: 0.39.0
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 4533
+
+env:
+   ND_SCANINTERVAL: 15m
```

Comment:
This looks to have 3 spaces in front, not 2

Reply:
Sorry. Using VScode YAML formatter now.